test/lit.cfg

```diff
@@ -135,8 +135,12 @@ current_llvm_version = "%s.%s" % (config.llvm_version_major,
                                   config.llvm_version_minor)
 config.available_features.add("llvm-" + current_llvm_version)
 for version in known_llvm_versions:
-   if version != current_llvm_version:
-      config.available_features.add("not-llvm-" + version)
+    if version != current_llvm_version:
```

Comment:
Sorry, I meant to be consistent with the original formatting (3 spaces). Besides leading to inconsistent formatting, it pollutes the git history with unnecessary changes (e.g lines 138-139).